+++ svelte.config.js	2022-08-10 23:52:52.874938600 -0400
@@ -5,11 +5,16 @@
 const config = {
 	// Consult https://github.com/sveltejs/svelte-preprocess
 	// for more information about preprocessors
-	preprocess: preprocess(),
+
+    preprocess: [
+        preprocess({
+            postcss: true,
+        }),
+    ],
 
 	kit: {
-		adapter: adapter()
-	}
+		adapter: adapter(),
+	},
 };
 
 export default config;
